docs/iap-integration/load_balancer_ingress.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
@djmailhot docs have yaml wrapped in shell commands, do you think we should keep this as is?  Or should we update to yaml?

Reply:
will update to yaml